Scripts/GameManager.gd

```diff
@@ -3,10 +3,38 @@ extends Node
 signal game_over(playerkilled)
 signal debug_mode_changed(previous, new)
 
+class Player:
+	var name: String
+	var color: Color
+	var input_code: String
+	var isDead: bool
+	var score: int
+	
+	func _init(name: String, color: Color, input_code: String) -> void:
+		self.name = name
+		self.color = color
+		self.input_code = input_code
+		self.isDead = false
+		self.score = 0
+
 var debug :bool setget set_debug_mode
 
 var is_game_over :bool
 
+var COLOR_SETS :PoolColorArray = [
+	Color(0,0,0,1), # Black
+	Color(1,0,0,1), # Red
+	Color(0,1,0,1), # Green
+	Color(0,0,1,1), # Blue
+	Color(1,1,0,1), # Yellow
+	Color(0,1,1,1), # Cyan
+	Color(1,0,1,1), # Magenta
+	Color(1,1,1,1), # White
+]
+
+## List of players
+var players = []
```

Comment:
I can't find how to set the type of the variable

Reply:
At max, we can define that as `var players :Array = []`